pgdog/src/backend/databases.rs

```diff
@@ -174,7 +174,7 @@ impl ToUser for (&str, Option<&str>) {
 pub struct Databases {
     databases: HashMap<User, Cluster>,
     manual_queries: HashMap<String, ManualQuery>,
-    mirrors: HashMap<String, Vec<Cluster>>,
+    mirrors: HashMap<String, Vec<(String, f32, usize)>>, // Vec<(destination, exposure, queue_depth)>
```

Comment:
You might want to `struct` that. Been  down this road before, even with different data types, things get confusing real quick.